common/src/main/java/dev/architectury/event/events/common/EntityEvent.java

```diff
@@ -130,4 +137,18 @@
          */
         void enterSection(Entity entity, int sectionX, int sectionY, int sectionZ, int prevX, int prevY, int prevZ);
     }
+    
+    interface AnimalTame {
+        /**
+         * Invoked before an tamable animal is tamed.
+         * This event only works on vanilla mobs. If needed in a modded entity, the mod should register this event on tame.
+         * Equivalent to Forge's {@code AnimalTameEvent} event.
+         *
+         * @param player The tamer.
```

Comment:
please order the parameters according to the actual parameters

Reply:
Done in 814fce1